imports/plugins/core/ui/client/components/button/buttonSelect.js

```diff
@@ -128,6 +128,7 @@ class ButtonSelect extends Component {
           {nonActiveButtons.map((button, key) => {
             return (<div
               className="button-item" key={key}
+              data-i18n={button.i18nKeyLabel}
```

Comment:
@mikemurray I am currently using a button component for the actual multi-function button. 
Do you mean I should use a button component for the dropdown?